scripts/configure-hypervisor.sh

```diff
@@ -371,7 +371,9 @@ generate_qemu_options() {
 		qemu_options+=(size:--disable-cloop)
 		qemu_options+=(size:--disable-dmg)
 		qemu_options+=(size:--disable-parallels)
-		qemu_options+=(size:--disable-vxhs)
+		if [ "${qemu_version_major}" -le 5 ] && [ "${qemu_version_minor}" -lt 1 ]; then
```

Comment:
This won't do what we want if the version is `4.2` for example. I'd use logic like that on #1151 as it's much clearer if you can use a "floating point version number" in the test.

Reply:
good catch, thanks!